extension/awsproxy/extension.go

```diff
@@ -15,15 +15,15 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/proxy"
 )
 
-type xrayProxy struct {
+type awsProxy struct {
 	logger *zap.Logger
 	config *Config
 	server proxy.Server
 }
 
-var _ extension.Extension = (*xrayProxy)(nil)
+var _ extension.Extension = (*awsProxy)(nil)
 
-func (x xrayProxy) Start(_ context.Context, host component.Host) error {
+func (x awsProxy) Start(_ context.Context, host component.Host) error {
```

Comment:
can we make this rename a separate chore PR?

Reply:
Sure, I was thinking about this too